### PR TITLE
Fix bug with pressure adjustment

### PR DESCRIPTION
**Pull request by eivindjahren**
For theoretical models of pressure adjustment, the order of properties returned by eval was not correct.

### CODE REVIEW

**Review by JiZur — APPROVED** [review]
Looks good to me

File: src/open_petro_elastic/config/model.py [hunk 1]
```diff
@@ -2,16 +2,15 @@
...
-
 from open_petro_elastic.material import Material, hashin_shtrikman_walpole
 from open_petro_elastic.material.sandstone import (
     friable_sand,
     murphy_coordination_number,
     patchy_cement,
 )
+from pydantic import parse_obj_as
+from pydantic.dataclasses import dataclass
+from typing_extensions import Literal
```
Comment (oyvindeide) [inline src/open_petro_elastic/config/model.py:13]: Why move these? According to the python overlords the previous order is the preferred.

**Review by oyvindeide — CHANGES_REQUESTED** [review]
Very tiny comment, others than that it looks good.

File: src/open_petro_elastic/config/model.py (hunk 1 above)
Comment (eivindjahren) [reply src/open_petro_elastic/config/model.py]: Fixed, just had isort configured incorrectly.

**Review by oyvindeide — APPROVED** [review]
LGTM!

**Pull request merged**